go.mod

```diff
@@ -22,4 +22,6 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776 // indirect
 )
 
+replace github.com/buildpacks/imgutil => github.com/jjbustamante/imgutil v0.0.0-20210817191209-1eb46031019f
```

Comment:
Do we still need it?

Reply:
Not sure if the [PR #126](https://github.com/buildpacks/imgutil/pull/126) on imgUtil repo was already merged, if it was, I will delete those lines

Reply:
You're right. I thought it was merged yesterday.

Reply:
I don't think so, the other PR was merged!

Reply:
Yeah, as I wrote, my mistake.

Reply:
@natalieparellano the imgutil is still not merged into main, the main was merged into the PR branch

Reply:
Whoops! Done now.

Reply:
At least I'm not the only one that thought it was already merged yesterday 😃